src/material-experimental/mdc-chips/chip.ts

```diff
@@ -382,7 +382,7 @@ export class MatChip extends _MatChipMixinBase implements AfterContentInit, Afte
           const isKeyboardEvent = event.type.startsWith('key');
 
           if (this.disabled || (isKeyboardEvent &&
-              this.HANDLED_KEYS.indexOf((event as KeyboardEvent).keyCode) !== -1)) {
+              this.HANDLED_KEYS.indexOf((event as KeyboardEvent).keyCode) === -1)) {
```

Comment:
nit: not super important for such a small set, but why not use `Set`?

Reply:
I don't see why not. Changed.